docs/dev/table/streaming/match_recognize.md

```diff
@@ -1038,7 +1038,7 @@ use [time attributes](time_attributes.html). To select those there are available
       <td><p>Returns the timestamp of the last row that was mapped to the given pattern.</p>
       <p>The resulting attribute is a <a href="time_attributes.html">rowtime attribute</a>
          that can be used in subsequent time-based operations such as
-         <a href="joins.html#interval-joins">interval joins</a> and <a href="#aggregations">group window or over
+         <a href="{% link joins.md#interval-joins %}">interval joins</a> and <a href="#aggregations">group window or over
```

Comment:
I guess this is an invalid link. Could you try to `build_docs.sh` in your local to verify this?

Reply:
Yes, @wuchong . You are right.
I'm so sorry for this mistake !
I corrected the invalid links based on your suggestions and tips of `build_docs.sh`.
Thank you for your reminding !